Khan/wonder-blocks
# [feature:internal] WB-618: Implement Noto in Wonder Blocks

## Summary
Continuing with the work done by @kevinbarabash in #296 

## Features
- Added `Armenian` to the list of supported fonts.
- Modified example to use `Dropdown` to switch between languages.

### Test plan
1. Open https://deploy-preview-452--wonder-blocks.netlify.com/#section-typography
2. Make sure the Noto fonts render correctly inside the `Noto for non-Latin languages` section.

![Screen Shot 2019-07-16 at 12 49 16 PM](https://user-images.githubusercontent.com/843075/61313287-28b8ac80-a7c8-11e9-9a6e-2721b3de1890.png)
